docker/pypi/wmagent-couchdb/Dockerfile

```diff
@@ -80,7 +57,7 @@ ENV PATH="/opt/couchdb/bin:/usr/local/bin/:${PATH}"
 ENV CRYPTOGRAPHY_ALLOW_OPENSSL_102=true
 
 # RUN <<EOF cat >> /opt/couchdb/.bashrc
-RUN <<EOF cat >> /home/$USER/.bashrc
+RUN <<EOF cat >> $HOME/.bashrc
```

Comment:
this I think will break in case somebody tries to build with an user other than the one, who is about to later run the container. Maybe escaping the `\$` sign before Home may help ... but I am still skeptical.....

It is really important, to my opinion, to have the command prompt correctly set (for both regular and root users inside the container). The best way how it was done was in the MariaDB container.

https://github.com/dmwm/CMSKubernetes/blob/326de33e3769e2f66e088c91093d3179a8d1cc12/docker/pypi/wmagent-mariadb/Dockerfile#L55-L64 

This must be set to all of the 3 containers the same way as  it was done there. And to cover both `regular user` and `root user` command prompts.

I do not know how this would work now. I did waste quite a lot of time in the near past to implement it in a generic way and to put it in `/etc/profile`, this way we will not depend on the $HOME valu to be resolved  at build time, but I have lost the change. Let me try to find it again, and come back to you with a concrete suggestion.

Reply:
ok, done